package.json

```diff
@@ -29,7 +29,7 @@
   },
   "devDependencies": {
     "babel": "^5.8.21",
-    "babel-eslint": "^4.1.3",
+    "babel-eslint": "^4.1.8",
```

Comment:
There was two issues I ran into that this fixed:

```
> testpilot@1.0.0 test:client /home/ubuntu/testpilot
> browserify -t babelify testpilot/frontend/static-src/test/**/*.js | testling | faucet
⨯ Error: ReferenceError: Can't find variable: Promise on line 28250
```
I couldn't quite find the source of this bug but searching points in the direction of an ES6 native promise bug. 

Also I ran into [this bug](https://github.com/babel/babel-eslint/issues/243) on previous tests. Updating `bable-eslint` seemed to have resolved both of them.